runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -17,21 +17,34 @@
 //! Runtimes implementing the v1 runtime API are recommended to forward directly to these
 //! functions.
 
+use sp_std::prelude::*;
 use primitives::v1::{
 	ValidatorId, ValidatorIndex, GroupRotationInfo, CoreState, GlobalValidationData,
 	Id as ParaId, OccupiedCoreAssumption, LocalValidationData, SessionIndex, ValidationCode,
 	CommittedCandidateReceipt, ScheduledCore, OccupiedCore, CoreOccupied, CoreIndex,
 	GroupIndex, CandidateEvent,
 };
 use sp_runtime::traits::Zero;
-use frame_support::debug;
+use frame_support::{
+	debug,
+	traits::Get,
+};
 use crate::{initializer, inclusion, scheduler, configuration, paras};
 
 /// Implementation for the `validators` function of the runtime API.
 pub fn validators<T: initializer::Trait>() -> Vec<ValidatorId> {
 	<inclusion::Module<T>>::validators()
 }
 
+/// Interface to the persistent (stash) identities of the current validators.
+pub struct ValidatorIdentities<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Is this used?

Reply:
No, this was used by `impl attestations::Trait` but since this runtime does not use them, this type is not used in this PR

Reply:
it's too late in the evening, now I see what you mean